### PR TITLE
[bug fix #255] found bug

### LINKED ISSUE #255: Possible bug / inconsistency in _GG.LOS_calc_signal()
While trying to finish / polish the PR #254 , and testing all tutorials, an error occured for:

tutorial plot_custom_emissivity.py
minimize = 'hybrid'
method = 'sum'

Upon closer inspection, it comes from line 3132 in _GG.py.
It seems method = 'sum' is the only case where the result is stored in sig_mv instead of sig.
I suspect this is the source of the error..
Indeed, there is no error if method  = 'simps' or method = 'romb'

The error-check matrix is

|      | sum | simps | romb |
|----|------|--------|-------|
hybrid | Error | ok | ok
calls | ok | ok | ok
memory | ok | ok | ok

The error message is:
```
Traceback (most recent call last):
  File "/Applications/Anaconda/python36/lib/python3.6/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Home/DV226270/ToFu_All/tofu_git/tofu/tofu/tests/tests09_tutorials/tests01_runall.py", line 182, in test03_plot_custom_emissivity
    self._test_tuto('tuto_plot_custom_emissivity')
  File "/Home/DV226270/ToFu_All/tofu_git/tofu/tofu/tests/tests09_tutorials/tests01_runall.py", line 164, in _test_tuto
    raise Exception(msg)
Exception: failed to get the current screen resources
Traceback (most recent call last):
  File "/Home/DV226270/ToFu_All/tofu_git/tofu/tuto_plot_custom_emissivity.py", line 92, in <module>
    t=time_vector)
  File "/Home/DV226270/ToFu_All/tofu_git/tofu/tofu/geom/_core.py", line 6066, in calc_signal
    Test=True,
  File "tofu/geom/_GG.pyx", line 3132, in tofu.geom._GG.LOS_calc_signal
TypeError: only size-1 arrays can be converted to Python scalars
```

@lasofivec can you have a look on a dedicated branch ?

### PR DESCRIPTION
**Pull request by lasofivec**
Fixes #255

### CODE REVIEW

File: tofu/geom/_core.py [hunk 1]
```diff
@@ -5815,7 +5817,7 @@ def check_ff(self, ff, t=None, ani=None):
         nt = len(t) if hasattr(t, '__iter__') else 1
 
         # .. Test anisotropic case .......................................
-        if ani is None:
+        if ani is False:
```
Comment (Didou09) [inline tofu/geom/_core.py]: I don't get it, if ani is False it means the user wants to force it to False, we should not change it.
We should only change it if None (means the user did not specify it).

File: tofu/geom/_core.py [hunk 2]
```diff
@@ -5969,7 +5971,7 @@ def calc_signal(
         self,
         func,
         t=None,
-        ani=None,
```
Comment (Didou09) [inline tofu/geom/_core.py]: Same comment as above, None means the user did not specify it, False means he wants to force it to False

Comment (lasofivec) [reply tofu/geom/_core.py]: updated

**Comment by lasofivec** [comment]
> Good, but:
> 
> * Could you find the reason why an error was thrown with vect=None ? It should not

Please see the lines where I tagged you

File: tofu/geom/_GG.pyx [hunk 3]
```diff
@@ -3129,7 +3128,7 @@ def LOS_calc_signal(func, double[:,::1] ray_orig, double[:,::1] ray_vdir, res,
                                                                 ray_orig[:, ii:ii+1],
                                                                 ray_vdir[:, ii:ii+1])
                     val_2d = func(pts, t=t, vect=-usbis, **fkwdargs)
-                    sig_mv[:, ii] = np.sum(val_2d, axis=-1)*loc_eff_res[0]
+                    sig[:, ii] = np.sum(val_2d, axis=-1)*loc_eff_res[0]
```
Comment (lasofivec) [inline tofu/geom/_GG.pyx]: @Didou09 :
> Could you find the reason why an error was thrown with vect=None ? It should not

It doesn't anymore. Here is the only real change in this PR, that fixes the bug.

Comment (lasofivec) [reply tofu/geom/_GG.pyx]: PS: if you see line 3319, it only happened if `ani=True, minimize=hybrid, method=sum`, and if you see line 2863 I took out the variable which was no longer need and verified all loops to see that this error was nowhere else.

File: tofu/tests/tests09_tutorials/tests01_runall.py [hunk 4]
```diff
@@ -154,8 +154,10 @@ def _test_tuto(cls, tuto=None, pathtuto=_HERE, root=_TFROOT):
             out = subprocess.run(cmd, shell=True, check=False,
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE)
-            if 'error' in out.stderr.decode().lower():
-                error = out.stderr.decode()
+            ls = out.stderr.decode().split('\n')
+            if any(['Error' in ss and not 'ModuleNotFoundError' in ss
+                    for ss in ls]):
+                error = '\n'.join(ls)
         except Exception as err:
```
Comment (lasofivec) [inline tofu/tests/tests09_tutorials/tests01_runall.py]: I was correcting this... thanks :)

**Pull request merged**